save-cli/build.gradle.kts

```diff
@@ -71,6 +86,13 @@ kotlin {
         dependsOn(linkReleaseExecutableTaskProvider)
     }
 
+    // disable building of some binaries to speed up build
+    // possible values: `all` - build all binaries, `debug` - build only debug binaries
+    val enabledExecutables = if (hasProperty("enabledExecutables")) property("enabledExecutables") as String else null
+    if (enabledExecutables != null && enabledExecutables != "all" || enabledExecutables == "debug") {
```

Comment:
Now I get: `Condition 'enabledExecutables == "debug"' is always false when reached`

I do not know how it was working previously...

Reply:
Do you set `-PenabledExecutables` to something other?

Reply:
> Do you set `-PenabledExecutables` to something other?

@petertrr Idea raises this warning without setting any parameters 
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/58667063/176409489-190c3c7a-53a0-4ba9-a90d-bf63162d06e2.png">

Reply:
Yep, because it's obviously a sub-case of the first part: `A != X || A == Y` 🤔 So, we are currently only distinguishing `-PenabledExecutables=all`